pontoon/base/templates/locale_selector.html

```diff
@@ -31,7 +31,7 @@
       <li class="clearfix{% if locale and locale == l %} current{% endif %}{% if l and l.chart %} filter{% endif %}">
         <span class="language {{ l.code|lower }}">{% if l and l.chart %}<a href="{% if project %}{{ url('pontoon.translate', l.code, project.slug) }}{% else %}{{ url('pontoon.locale', l.code) }}{% endif %}" class="clearfix">{% endif %}{{ l.name }}{% if l and l.chart %}</a>{% endif %}</span>
         <span class="code">{% if l and l.chart %}<a class="code" href="{% if project %}{{ url('pontoon.translate', l.code, project.slug) }}{% else %}{{ url('pontoon.locale', l.code) }}{% endif %}" class="clearfix">{% endif %}{{ l.code }}{% if l and l.chart %}</a>{% endif %}</span>
-        {% if not locale %}
+        {% if not locale and l.chart %}
```

Comment:
I've become the thing I hated the most. We should not be using the existence of `l.chart` as a flag that the locale is not active on the project.

But these pages are in my sights as the first ones to be rewritten, so WHO CARES